refinery/tool_manager/models.py

```diff
@@ -298,6 +298,11 @@ class Tool(OwnableResource):
     )
     tool_launch_configuration = models.TextField()
     tool_definition = models.ForeignKey(ToolDefinition)
+    display_name = models.CharField(
+        max_length=250,
+        unique=True,
```

Comment:
Unique names per tool? Is this is necessary or wanted? @ngehlenborg @mccalluc  Thoughts? How are we error handling this (what would be the api response)?

Reply:
User-supplied `display_names` are to be unique per @ngehlenborg's request.
The api response will be a 400 error with the message: `A Tool already exists with a display_name of: '<tool_display_name>'`